editor/block-mover/mover-label.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Wordpress dependencies
+ */
+import { __, sprintf } from 'i18n';
+
+/**
+ * Return a label for the block movement controls depending on block position.
+ *
+ * @param  {number}  selectedCount Number of blocks selected.
+ * @param  {string}  type          Block type - in the case of a single block, should
+ *                                 define its 'type'. I.e. 'Text', 'Heading', 'Image' etc.
+ * @param  {number}  firstIndex    The index (position - 1) of the first block selected.
+ * @param  {boolean} isFirst       This is the first block.
+ * @param  {boolean} isLast        This is the last block.
+ * @param  {number}  dir           Direction of movement (> 0 is considered to be going
+ *                                 down, < 0 is up).
+ * @return {string}                Label for the block movement controls.
+ */
+export function getBlockMoverLabel( selectedCount, type, firstIndex, isFirst, isLast, dir ) {
+	const position = ( firstIndex + 1 );
+
+	if ( selectedCount > 1 ) {
+		return getMultiBlockMoverLabel( selectedCount, firstIndex, isFirst, isLast, dir );
+	}
+
+	if ( isFirst && isLast ) {
+		// translators: %s: Type of block (i.e. Text, Image etc)
```

Comment:
Have left as-is for now, but if it is decided that the comment should need updated, it's a tiny change and I'm happy to make it.